### ISSUE TITLE
Question about the performance of ANIL

### ISSUE THREAD

**Issue by peterdarkdarkgogo** [huaxiuyao/ATS#1]
Hi, I try to extract the ANIL module and ran the experiment(5way-1shot) based on the dataset miniimagenet, but the test set performance is just 44, which not matched with the original paper performance 46.7+-0.4.

Would you mind sharing some advice ? Thank you.

![2331638536266_ pic_hd](https://user-images.githubusercontent.com/94344227/144609363-f3634c00-47da-40fe-b2a3-c06530dd5d84.jpg)

**Comment by wangyu-ustc** [comment]
I have attached the result that I have run. This is the performance we have reported in the paper. The code has no problem. 

According to my experiences, results from implementations of various versions could differ up to 1-2 percent, which could be attributed to random seeds. So, I suggest you have another run with a different seed and check the results.

![image](https://user-images.githubusercontent.com/44958517/144739281-567ddd5d-5d38-4d21-97e4-c17953dac61f.png)
